docs/modules/ROOT/pages/advanced-how-tos/releasing/tenant-release-pipelines.adoc

```diff
@@ -75,6 +77,78 @@ kubectl apply -f ReleasePlan.yaml -n dev
+
+*Example tenant pipeline*
+
+[source,yaml]
+----
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: my-tenant-pipeline
+spec:
+  params:
+    - name: release <.>
+      type: string
+    - name: releasePlan <.>
+      type: string
+    - name: snapshot <.>
+      type: string
+  tasks:
+    - name: echo-resources
+      taskSpec:
+       steps:
+         - name: echo resources
+           image: ubuntu:latest
+           script: |
+             #!/usr/bin/env sh
+             echo "Release $(params.release)"
+             echo "ReleasePlan $(params.releasePlan)"
+             echo "Snapshot $(params.snapshot)"
+----
+<.> Namespacedname to the Release populated automatically by the release service (eg. dev-workspace/my-tenant-release).
+<.> Namespacedname to the ReleasePlan populated automatically by the release service (eg. dev-workspace/publish).
+<.> Namespacedname to the Snapshot populated automatically by the release service (eg. dev-workspace/my-snapshot).
+
+If you write a good reusable release pipeline, please submit it to our link:https://github.com/konflux-ci/release-service-catalog[catalog] so others can use it.
+
+== Final pipeline
+
+Another type of tenant pipeline runs at the end of the release workflow. This is known as the final pipeline, and it allows you to execute a pipeline after the tenant or managed pipeline has completed.
+
+You can use this pipeline, for example, to send Slack notifications once your images have been pushed or to generate a changelog summarizing the new changes.
+
+To enable it, modify the ReleasePlan by adding the `finalPipeline` field.
+
+*Example of final pipeline declaration*
+
+[source,yaml]
+----
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ReleasePlan
+...
+spec:
+  ...
+  finalPipeline:
+    pipelineRef: <.>
+      resolver: git
+      params:
+        - name: url
+          value: "https://github.com/<your-github-user>/<your-pipeline-repo>.git"
+        - name: revision
+          value: main
+        - name: pathInRepo
+          value: "<path-to-your-pipeline>"
+    serviceAccountName: appstudio-pipeline <.>
+----
+<.> Reference to the tenant pipeline to be executed in the development workspace.
+<.> The name of the service account used to execute the tenant pipeline.
+
+Both tenant and final pipelines receive the same parameters (e.g., release, releasePlan, and snapshot), allowing them to be used interchangeably. The key difference is that the final pipeline runs at the end of the release workflow, meaning the release status will contain the final outcome and all generated artifacts.
```

Comment:
Is it exactly `release, releasePlan, and snapshot` or are there more? I think there are more, right? The reason I'm asking is if in fact you mean "i.e." and not "e.g.". But I think there are more - like the git parameters. So in that case it's ok.

Reply:
It is only those, so yes, I mean i.e. Fixed.

Reply:
Thanks.